taworks/taform/views.py

```diff
@@ -336,10 +336,10 @@ def load_url(request, hash):
     url = get_object_or_404(models.Course, url_hash=hash)        
     courses = models.Course.objects.filter(url_hash=hash)
     course_id = courses[0].id
-    apps = models.Application.objects.filter(course_id=course_id).exclude(preference=0).order_by('student__first_name')
+    apps = models.Application.objects.filter(course_id=course_id).exclude(preference=0)
```

Comment:
"It’s not a problem in the writing back, it’s a problem in the pulling from
If it orders by first name, it pulls the same student twice as opposed to the two unique students"

just to recap I'm not sure that's the reason because if you look at my original video, the reasons for the bugged person not submitting ranking they had separate reasonings.

if it was pulling from the same student twice then the second student wouldn't have been displayed at all right?